app/routes/admin/settings/images.js

```diff
@@ -5,10 +5,15 @@ export default Route.extend({
     return this.get('l10n').t('Images');
   },
 
-  model() {
-    return this.get('store').query('image-size', {
-      sort: 'id'
-    });
+  async model() {
+    return {
+      speaker: await this.get('store').queryRecord('speaker-image-size', {
```

Comment:
the names seems counter-intuitive, it's an image size, but seems to represent an actual speaker.